README.md

```diff
@@ -1,95 +0,0 @@
-# Лабораторные работы для 2-го курса ФПЛ (2023/2024)
```

Comment:
Why did you remove it? Good useful file...

Reply:
I removed it by accident, you helped me to fix it in class. The file is very useful, i agree